packages/carbon-web-components/src/components/button/button-story.ts

```diff
@@ -9,177 +9,320 @@
+export const Secondary = (args) => {
+  const {
+    disabled,
+    href,
+    size,
+    tooltipAlignment,
+    tooltipPosition,
+    tooltipText,
+    onClick,
+  } = args?.[`${prefix}-btn`] ?? {};
   return html`
-    <cds-btn-skeleton
-      ?disabled=${disabled}
-      size=${ifDefined(size)}
-      href=${ifDefined(href || undefined)}
-      ?isExpressive="${isExpressive}"
-      @click=${onClick}>
-    </cds-btn-skeleton>
+    <cds-btn
+      type="secondary"
+      ?disabled="${disabled}"
+      href="${ifDefined(href)}"
+      size="${ifDefined(size)}"
+      @click=${onClick}
+      tooltip-text="${tooltipText}"
+      tooltip-alignment="${tooltipAlignment}"
+      tooltip-position="${tooltipPosition}">
+      Button
+    </cds-btn>
   `;
 };
 
-skeleton.parameters = {
-  percy: {
-    skip: true,
-  },
-  knobs: {
-    [`${prefix}-btn-skeleton`]: () => ({
-      kind: select('Button kind (kind)', kinds, BUTTON_KIND.PRIMARY),
-      disabled: boolean('Disabled (disabled)', false),
-      size: select('Button size (size)', sizes, null),
-      href: textNullable('Link href (href)', ''),
-      onClick: action('click'),
-      isExpressive: boolean('expressive (isExpressive)', false),
-    }),
-  },
+export const SetOfButtons = (args) => {
+  const {
+    disabled,
+    href,
+    size,
+    tooltipAlignment,
+    tooltipPosition,
+    tooltipText,
+    onClick,
+  } = args?.[`${prefix}-btn`] ?? {};
+  return html`
+    <cds-btn-set>
+      <cds-btn
+        type="secondary"
+        ?disabled="${disabled}"
+        href="${ifDefined(href)}"
+        size="${ifDefined(size)}"
+        @click=${onClick}
+        tooltip-text="${tooltipText}"
+        tooltip-alignment="${tooltipAlignment}"
+        tooltip-position="${tooltipPosition}">
+        Secondary button
+      </cds-btn>
+      <cds-btn
+        type="primary"
+        ?disabled="${disabled}"
+        href="${ifDefined(href)}"
+        size="${ifDefined(size)}"
+        @click=${onClick}
+        tooltip-text="${tooltipText}"
+        tooltip-alignment="${tooltipAlignment}"
+        tooltip-position="${tooltipPosition}">
+        Primary button
+      </cds-btn>
+    </cds-btn-set>
+  `;
+};
+
+export const skeleton = () => {
+  return html` <cds-btn-skeleton> </cds-btn-skeleton> `;
+};
+
+export const Tertiary = (args) => {
+  const {
+    disabled,
+    href,
+    size,
+    tooltipAlignment,
+    tooltipPosition,
+    tooltipText,
+    onClick,
+  } = args?.[`${prefix}-btn`] ?? {};
+  return html`
+    <cds-btn
+      type="tertiary"
+      ?disabled="${disabled}"
+      href="${ifDefined(href)}"
+      size="${ifDefined(size)}"
+      @click=${onClick}
+      tooltip-text="${tooltipText}"
+      tooltip-alignment="${tooltipAlignment}"
+      tooltip-position="${tooltipPosition}">
+      Button
+    </cds-btn>
+  `;
 };
 
 export default {
   title: 'Components/Button',
   parameters: {
     ...storyDocs.parameters,
+    knobs: {
+      [`${prefix}-btn`]: () => ({
```

Comment:
some of the knobs are missing from the controls in https://react.carbondesignsystem.com/?path=/story/components-button. if possible can we match not used knobs that are the same in react too?

Reply:
I added a couple more knobs that I had missed before, but can you clarify what you mean with the not used knobs?

Reply:
hey yeah sorry meant to come back and add! i think now just missing `isExpressive` and maybe `isSelected` for ghost variant

Reply:
Weird, seems like `isSelected` does nothing? 
https://react.carbondesignsystem.com/?path=/story/components-button--ghost&args=isSelected:true;

Reply:
https://github.com/carbon-design-system/carbon/blob/9cc6a01fe11636dfb14093a874eb1c0c2b08ca83/packages/react/src/components/Button/Button.js#L97

i think not always the controls they have setup work as expected.... maybe we don't need it as a knob, but definitely need it as a prop. Could we possibly add isExpresive as a knob and then get all of this in!!! lol